### PR TITLE
PI-727: Requests routed with CF\Router\RequestRouter now.  Plugin API…

### PR DESCRIPTION
**Pull request by jwineman**
… logic moved to cloudflare/cloudflare-plugin-backend.

### CODE REVIEW

**Comment by jwineman** [comment]
This PR is to update the plugin to be compliant with https://github.com/cloudflare/cloudflare-plugin-backend/pull/10

File: Backend/PluginActions.php [hunk 1]
```diff
@@ -126,8 +61,6 @@ public function patchPluginSettingsDefaultSettings() {
         $this->clientAPI->callAPI(new \CF\API\Request('PATCH', 'zones/'. $zoneId .'/settings/mirage', array(), array('value' => 'off')));
 
         $this->clientAPI->callAPI(new \CF\API\Request('PATCH', 'zones/'. $zoneId .'/settings/rocket_loader', array(), array('value' => 'off')));
-
-        return $this->api->createAPISuccessResponse($this->api->createPluginResult(\CF\API\Plugin::SETTING_DEFAULT_SETTINGS, true, true, ''));
```
Comment (thellimist) [inline Backend/PluginActions.php]: Why did you delete the api response?

Comment (jwineman) [inline Backend/PluginActions.php]: The response is returned in cloudflare/cloudflare-plugin-backend CF\API\AbstractPluginsActions.php now.
